### PR TITLE
Remove Cypress #1

### PR DESCRIPTION
**Pull request by arelra**
## What does this change?

Remove Cypress code from the repository

There are still some follow up items which I thought best to raise as separate PRs so they are easier to review and not lost in the noise:

- update `README.md`
- update `testing.md`
- update `SignInGate/README.md`
- converting the skipped `SetABTests` test to a react-testing-library test

## Why?

We have migrated to Playwright

### CODE REVIEW

File: dotcom-rendering/playwright/lib/locators.ts [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
+const TEN_SECONDS = 10_000;
+
 const expectToBeVisible = async (
 	page: Page,
 	selector: string,
 	nth = 0,
 ): Promise<void> => {
 	await expect(page.locator(selector).nth(nth)).toBeVisible({
-		timeout: 10000,
+		timeout: TEN_SECONDS,
```
Comment (arelra) [inline dotcom-rendering/playwright/lib/locators.ts:12]: Drive by

File: dotcom-rendering/src/model/enhance-switches.ts [hunk 2]
```diff
@@ -1,6 +1,15 @@
 import type { Switches } from '../types/config';
 
-export type ABTestSwitches = { [key: `ab${string}`]: boolean | undefined };
+type ABTestSwitches = { [key: `ab${string}`]: boolean };
+type ABTestSwitchesKey = keyof ABTestSwitches;
+type ABTestSwitchesValue = ABTestSwitches[ABTestSwitchesKey];
+
+const isABTestEntry = (
```
Comment (arelra) [inline dotcom-rendering/src/model/enhance-switches.ts:7]: Added a type guard to the filter so the correct type is evaluated here

File: dotcom-rendering/src/model/enhance-switches.ts [hunk 3]
```diff
@@ -1,6 +1,15 @@
...
+	entry: [string, boolean | undefined],
+): entry is [ABTestSwitchesKey, ABTestSwitchesValue] => {
+	const [key, value] = entry;
+	return key.startsWith('ab') && typeof value === 'boolean';
```
Comment (mxdvl) [inline dotcom-rendering/src/model/enhance-switches.ts:11]: Do we have an `isBoolean` helper from `@guardian/libs`?

**Review by mxdvl — APPROVED** [review]
`Cy` you later!

**Pull request merged**